pipelines/rj_smas__disparo_pic/tasks.py

```
@@ -0,0 +1,310 @@
+        batch_payload = dispatch_payload.copy()
+        batch_payload["destinations"] = batch
+        batch_payload["campaignName"] = f"{original_campaign_name}-{dispatch_date[:10]}-lote{i}"
+
+        log(f"Disparando lote {i} de {total_batches} com {len(batch)} destinos")
+
+        response = api.post(path=f"/callcenter/hsm/send/{id_hsm}", json=batch_payload)
+
+        if response.status_code != 201:
+            log(f"Falha no disparo do lote {i}: {response.text}")
+            response.raise_for_status()
+            raise Exception(f"Dispatch failed: {response.text}")
```

Ruff (0.14.1)

Create your own exception

(TRY002)